iron-selectable.html

```diff
@@ -46,8 +46,11 @@
     properties: {
 
       /**
-       * If you want to use the attribute value of an element for `selected` instead of the index,
-       * set this to the name of the attribute.
+       * If you want to use an attribute value or property of an element for
+       * `selected` instead of the index, set this to the name of the attribute
+       * or property. The attribute or property name **must be given in
```

Comment:
I think the comment is a bit of a lie :) You can still use camel case, right? Otherwise it would be a breaking change?